fix promtheus and grafana setup (#592)
harness/developer-hub: docs/platform/2_Delegates/manage-delegates/delegate-metrics.md

This document explains how to configure the Prometheus monitoring tool for metrics collection, and how to configure the Grafana analytics tool for metrics display. This document includes example YAML you can use to create application manifests for both configurations.

### Apply the prometheus.yml file

> Take the below example for prometheus.yaml, this install prometheus workload (harness-delegate-prometheus-deployment) and a service (harness-delegate-prometheus-service) in your kubernetes cluster. This will also create a load balancer with ip for you to access the prometheus UI

Reviewer: Apply the following prometheus.yml file to install a Prometheus workload (harness-delegate-prometheus-deployment) and a service (harness-delegate-prometheus-service) in your Kubernetes cluster. This manifest also creates a load balancer with an IP address you can use to access the Prometheus UI.

   kubectl apply -f grafana.yml
   ```
   

> 4. This will also create a load balancer and service in kubernetes cluster. Click on the exposed URL to access grafana 

Reviewer: ```suggestion
4. This manifest also creates a load balancer and service in your Kubernetes cluster. Click the exposed URL to access Grafana.
```
